create-trust-store.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# Copyright 2022 Hewlett Packard Enterprise Development LP
+
```

Comment:
Is this running from within the context of the Nexus pod shell?  If so I don't think these messages will appear in the pod log.

I think you will need to add something similar to:
`log() {
  LOG=/proc/1/fd/1
  if [[ ! -z "$1" ]]; then echo "$(date) [$0] $1" | tee $LOG; fi
}`

Reply:
This runs from the initContainer, not the nexus pod.